scripts/west_commands/runners/openocd.py

```diff
@@ -167,8 +178,9 @@ def do_flash(self, **kwargs):
                                 '-c', 'reset halt'] +
                ['-c', self.verify_cmd + ' ' + hex_name] +
                post_verify_cmd +
-               ['-c', 'reset run',
-                '-c', 'shutdown'])
+               ['-c', 'reset run'] +
+               clear_halt_debug_en_cmd +
```

Comment:
This name is confusing mechanism with policy. Please rename it to `post_reset_cmd` instead.

Reply:
Yeap, that's fine.